scripts/build-deb.sh

```diff
@@ -52,15 +52,14 @@ build-deb () {
         setup.cfg \
         setup.py \
         ulauncher \
-        ulauncher.desktop.in \
+        ulauncher.desktop \
         $tmpsrc \
         --exclude-from=.gitignore
 
     # This is only needed because data/preferences is in .gitignore
     cp -r data/preferences $tmpdir/data/preferences
 
     cd $tmpsrc
-    sed -i "s/%VERSION%/$version/g" setup.py
```

Comment:
The version passed into this script is used for the changelog for the debian package, but has no relationship to the actual embedded version string unless they happen to match or be the same.

Reply:
I would rather keep it like this for now since it's still an improvement from before. The scripts are working, but have much room for removing duplication and unnecessary complications, but if I start changing this I have to change it in all files and, documentation, and the wiki, and make changes in files I never ran or can't run like make-release.

Reply:
This inconsistency was already there before and outside of the scope of the PR, but fixed in 7bf0805 now anyway